src/containers/WalletBackupContainer.js

```diff
@@ -51,7 +58,7 @@ class WalletBackupContainer extends PureComponent<TWalletBackupContainerProps, {
       })
     }
 
-    onLogin()
+    this.login()
```

Comment:
Should be this.props.login() or login() if above props destructuring will be changed appropriately